sandbox/tests/test scenes/many light sampling/4_colored_point_lights.appleseed

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- File generated by appleseed.so version 1.7.0-beta (SSE 4.2). -->
+<project format_revision="18">
+    <scene>
```

Comment:
No, it was your initial scene with plane right next to it. I removed it completely now as we don't use it any more and have the generator for it.